lib/chef/provider/script.rb

```diff
@@ -18,6 +18,7 @@
 
 require "tempfile"
 require "chef/provider/execute"
+require "chef/win32/security" if Chef::Platform.windows?
```

Comment:
this is actually correct because it happens so early that ohai will not have run, there can't be a node object, and chefspec shouldn't require this file either.